GoogleForCreators/web-stories-wp
# Editor: Border Radius Support For Text Highlight Mode

## Summary
Applies the border radius to the highlight elements in text highlight mode.

## Relevant Technical Choices
Had to use a different method for applying border radius than the existing one because lines can have various widths. Approach uses px values for borer radius in the editor space. so should be consistent and responsive to story size.

## To-do
NA

## User-facing changes
- Corner radius should now apply to the highlights in text highlight mode
- All text elements should now come in with a default border radius of 2.

## Testing Instructions

-> go to edit story
-> add text to the story
-> see that there's a default border radius of 2
-> change the text's fill mode to highlight
-> see that the border radius applies to the highlight (should also be there while editing text and in the preview)
-> play with the border radius and see that all changes are applying to the highlighted lines

---



Fixes #5715
